src/components/Cards/item.tsx

```diff
@@ -7,20 +7,20 @@ export interface IItem {
   cardIcon?: EIconProps;
   label?: string;
   title: string;
-  text: string;
-  link?: {
+  text?: string;
+  links?: Array<{
```

Comment:
With this you are breaking the backward compatibility.

So, follow the [semver](https://semver.org/) guidelines to bump the proper version